### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by martpie** [martpie/museeks#436]
Make the app fully and completely accessible.

- [x] Make use of `jsx-a11y`
- [ ] Test visual accessibility
- [ ] Test keyboard-only scenario

**Comment by vprigent** [comment]
Opquast kind of accessible ?

**Comment by KeitIG** [comment]
I don't know what this is 😄 Accessible kind of accessible. 

Screen-reader + full keyboard support should be a good start.

**Comment by vprigent** [comment]
Opquast references a list of good practices regarding the web in general with a strong emphasis on accessibility. Making sure we have a color-blind option or that the contrast makes things readable to everyone is part of accessibility but often forgotten.

**Comment by KeitIG** [comment]
I will need to check contrast, but the new (now old) dark theme improved things significantly.

**Comment by KeitIG** [comment]
So I changed the eslint config so we start use `jsx-a11y`, it is quite funny to see it will achieve a good part of the goal of this issue when I will be done with all the errors and warnings.
